recipes/libtiff/all/conanfile.py

```diff
@@ -105,15 +105,19 @@ def source(self):
                   destination=self._source_subfolder, strip_root=True)
 
     def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
         # Rename the generated Findjbig.cmake and Findzstd.cmake to avoid case insensitive conflicts with FindJBIG.cmake and FindZSTD.cmake on Windows
         if self.options.jbig:
             tools.rename(os.path.join(self.build_folder, "Findjbig.cmake"),
                          os.path.join(self.build_folder, "ConanFindjbig.cmake"))
+        else:
+            os.remove(os.path.join(self.build_folder, self._source_subfolder, "cmake", "FindJBIG.cmake"))
```

Comment:
If upstream CMakeLists still calls `find_package(JBIG)` when jbig is disabled, there is something wrong we should fix.

Actually I've tested with all dependencies disabled, libtiff still tries to detect at least jbig, zstd, zlib and lzma (xz_utils), and it may find system libs. zstd detection fails for the same reason than jbig.